cpi-api-v1.html.md.erb

```diff
@@ -14,12 +14,19 @@ resource_pools:
   cloud_properties: {instance_type: r3.8xlarge}
 ```
 
+For an overview of the sequence of CPI calls, the following resources are helpful:
+- [bosh components](bosh-components.html) and its example component interaction during new vm creation
+- typical vm instanciation by bosh-init described in the [bosh-init architecture doc](https://github.com/cloudfoundry/bosh-init/blob/master/docs/architecture.md#deploy-command-flow) where calls to the cpi are mentionned as `cloud.` 
+
 ---
 ## <a id="stemcells"></a> Stemcell management
 
 ### <a id="create-stemcell"></a> `create_stemcell`
 
-Creates a reusable VM image in the IaaS from the stemcell image. It's used later for creating VMs. For example AWS CPI creates an AMI and returns AMI ID.
+Creates a reusable VM image in the IaaS from the [stemcell](stemcell.html) image. It's used later for creating VMs. For example AWS CPI creates an AMI and returns AMI ID. 
+Stemcell tarballs are currently specific to each Iaas/CPI (e.g. include cloud-init or hypervisor specific drivers). In the future, stemcells will become [universal](https://www.pivotaltracker.com/epic/show/927178) and CPI will bring in the Iaas specific bits before creating the VM image. 
```

Comment:
It's great that there exists an epic for that - however, it doesn't seem to contain any stories at all. Also, https://github.com/cloudfoundry/bosh-notes doesn't contain anything about universal stemcells. If it is that unspecific at the current point in time - should we leave that out of here?